javascript/javascript3/week2/yesNo.js

```diff
@@ -0,0 +1,180 @@
+const movies = fetch('https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json')
+.then(response=>response.json())
+   .then(data => data.map(badmovies => badmovies).filter(movieList => movieList.rating < 6 && movieList.year >= 2000));
+   console.log(movies);
```

Comment:
Here `movies` takes the result of a chain of promises which is in the end a `Promise` instance as well. What we're printing here with `console.log` isn't the data but the `Promise` instance.
What we need to do is call `console.log` within a `.then` code block after fetching the movies.

Another thing to point out is we're required to print two arrays of movies: one with bad movies and one with bad movies after 2000. A suggestion is to chain a couple of `.then` calls to filter the movies accordingly and print them :)